frontend/src/components/common/action-button/ActionButton.tsx

```diff
@@ -1,14 +1,16 @@
-import React, { useState } from 'react'
+import { useState } from 'react'
 import cx from 'classnames'
 
 import styles from './ActionButton.module.scss'
 
+import type { MouseEvent as ReactMouseEvent } from 'react'
```

Comment:
Let's move this up top together with the react import, combine them if possible?